client/src/app/services/manage-members.service.ts

```diff
@@ -15,4 +15,9 @@ export class ManageMembersService {
   getAllClients(): Observable<any> {
     return this.http.get<User[]>(`${this.BASEURL}/getAllClients/`);
   }
+
+  sendNameToObtainClients(name): Observable<any>
+  {
+    return this.http.post('http://localhost:8000/getSomeClients/', JSON.stringify(name));
```

Comment:
Don't hard code the URL inside the request, create a BASEURL variable on top where you specify the global URL used. you can look at how it's done inside the other services

Reply:
fixed in commit 38fa446370aa1c6aa2e3d4df0f78677c0e8076b2